index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <div id="Form">
+
+    
+    <form action="" id="survey-form">
+        <div>
+            <label id="name-label" for="name" >Name</label><br>
+            <input type="text" id="name" required placeholder="enter you name" class="large">
+        </div>
+        <div>
+            <label id="email-label" required for="email">Email</label><br>
+            <input type="email" required id="email" placeholder="enter you email" class="large">
+        </div>
+        <div>
+            <label id="number-label" for="number">Age(optional)</label><br>
+            <input type="number" id="number" min="10" max="90" placeholder="entre you age" class="large">
```

Comment:
Give the classes MeaningFul Names

Reply:
Kindly explain why the student needs to give meaningful names to the classes. This will help the student to better understand why it has to be done, hence it will be easier to implement it. :+1: